drv/opentitan-aontimer/Cargo.toml

```diff
@@ -0,0 +1,24 @@
+[package]
+name = "drv-ot-aontimer"
+version = "0.1.0"
+edition = "2021"
+
+[dependencies]
+userlib = {path = "../../sys/userlib", features = ["panic-messages"]}
+zerocopy = "0.6.1"
+
+[features]
+semihosting-riscv = [ "riscv-semihosting", "userlib/log-semihosting" ]
+log-null = ["userlib/log-null"]
+
+[target.'cfg(target_arch = "riscv32")'.dependencies]
```

Comment:
Wouldn't this be always true?

Reply:
For this hardware peripheral we know it will only ever be on riscv, but hypothetically you could have an equivalent behaving arm peripheral.  The driver I think is otherwise arch agnostic and if complied for arm, these crates are not needed.
This is all a stretch though, just throwing ideas out there.  Alternatively, we could add a compile time check to prevent this driver from being compiled outside riscv, then just always include them.

Reply:
I think we can leave it in here. I was just curious what strategy we want to follow for writing the `Cargo.toml` files. We might see a lot of crates that probably would be used for RV only.